share/tools/web_config/sample_prompts/robbyrussell.fish

```diff
@@ -7,8 +7,8 @@ function fish_prompt
         set -g __fish_robbyrussell_functions_defined
         function _git_branch_name
             set -l branch (git symbolic-ref --quiet HEAD ^/dev/null)
-            if test $branch
-                echo $branch | sed -e 's|^refs/heads/||'
+            if set -q branch[1]
+                echo (string sub -s 12 $branch)
```

Comment:
This is better than executing the external `sed` command but hardcoding the string length in this manner is fragile and not obvious. By "not obvious" I mean that the value of "12" is not obviously related to the "refs/heads/" prefix. Unless you use a construct like `(math 1 + (string length "refs/heads/"))`. But it is simpler to use `string replace -r '^refs/heads/' '' $branch`.